greenboot.spec

```diff
@@ -63,6 +63,13 @@ Requires:           %{name} = %{version}-%{release}
 %description reboot
 %{summary}.
 
+%package update_platforms_check
```

Comment:
the sub-package name will be `greenboot-update_platforms_check` , maybe change the underscore to dash to keep in line with other sub-package names?

Reply:
Definitely